### PR TITLE
Mentioned non_field_errors in form validation docs.

### PR DESCRIPTION
**Pull request by sks444**

### CODE REVIEW

**Comment by sks444** [comment]
Related to the discussion with @adamchainz at [Django Forum](https://forum.djangoproject.com/t/forms-validationerror-doesnt-print-any-error-message-to-the-template/1783/).

**Comment by adamchainz** [comment]
Title should be past tense. Suggestion: "Mentioned non_field_errors in Form.clean() docs"

File: docs/ref/forms/validation.txt [hunk 1]
```diff
@@ -366,6 +366,9 @@ example::
 
 In this code, if the validation error is raised, the form will display an
 error message at the top of the form (normally) describing the problem.
+Note that errors generated here will be form errors and not field errors,
+to display these errors you'll have to define ``{{ form.non_field_errors }}``
+in your template.
```
Comment (adamchainz) [inline docs/ref/forms/validation.txt:371]: I edited down but don't think I lost any information:

```suggestion
Such errors are non-field errors, which are displayed in the template with ``{{ form.non_field_errors }}``.
```

Comment (sks444) [reply docs/ref/forms/validation.txt]: Thanks @adamchainz, this is much better. :)

**Comment by felixxm** [comment]
@sks444 Thanks :+1:

**Comment by felixxm** [comment]
Merged in ab903fe304c347e669024ec482832df5f4fd869a.

**Pull request closed without merging**